deploy/Dockerfile

```
@@ -0,0 +1,22 @@
+FROM golang:alpine as builder
+MAINTAINER Jack Murdock <jack_murdock@comcast.com>
+
+WORKDIR /go/src/github.com/comcast/codex-fenrir
+
+RUN apk update && apk upgrade && \
+    apk add --no-cache bash git openssh
+
+COPY . .
```

since this one is non local it should run mod vendor
```suggestion
COPY . .
RUN GO111MODULE=on go mod vendor
```